.github/workflows/cleanup.yaml

```diff
@@ -0,0 +1,33 @@
+name: Branch Cleanup
```

Comment:
Caches are already automatically deleted. We struggle with the API requests we are allowed to make. We tried to do a similar thing on core, but it caused us to go over the limits. I would omit this for now.

Reply:
Do you want to commit it and just disable it for now?

Reply:
Yeah, remove it for now. We rely on the automatic eviction/LRU that happens already.

Reply:
Clarify please.  You said yeah to my question (commit and disable), but did you mean I should remove it here and stash locally just in case?

Reply:
The cleanup.yaml should not be used at this point. Adding dead code to a repo is not something we generally should do. Please remove the file.

If you want to keep it around in a local stash or branch, that this fine. But right now, we should not have this in our repo.

../Frenck